source/install/cloud-changelog.md

```diff
@@ -21,25 +28,69 @@ Latest Mattermost Cloud releases:
  - Cloud Professional monthly will no longer be offered to new customers starting February 2, 2023.
```

Comment:
*February 23

Reply:
I put March 2nd which is when the next release is scheduled.

Reply:
Good catch -- thanks @amyblais